op-supervisor/supervisor/frontend/frontend.go

```diff
@@ -85,3 +85,8 @@ func (a *AdminFrontend) Stop(ctx context.Context) error {
 func (a *AdminFrontend) AddL2RPC(ctx context.Context, rpc string, jwtSecret eth.Bytes32) error {
 	return a.Supervisor.AddL2RPC(ctx, rpc, jwtSecret)
 }
+
+// Rewind removes some L2 chain data from the supervisor backend, starting from the given block.
+func (a *AdminFrontend) Rewind(ctx context.Context, chain eth.ChainID, block eth.BlockID) error {
+	return a.Supervisor.Rewind(ctx, chain, block)
```

Comment:
Might be worth a log message here to note that the rewind was requested from the API? I know we'll have debug logging to show the API call but that often won't show up and this is a pretty significant admin action to take.

Reply:
The entire frontend structure lacks a logging structure unfortunately. In the interest of keeping this PR compact, I've added a todo with a matching github ticket.